server/events/command_runner.go

```diff
@@ -202,6 +202,13 @@ func (c *DefaultCommandRunner) RunCommentCommand(ctx context.Context, baseRepo m
 	}
 
 	if err := c.PreWorkflowHooksCommandRunner.RunPreHooks(ctx, cmdCtx); err != nil {
+		// Replace approve policies command with policy check if preworkflow hook fails since we don't use
+		// approve policies statuses and a consecutive approve_policies command would only set the policy checks status
+		// to success, not set the failed approve_policies to success.
+		if cmd.Name == command.ApprovePolicies {
+			cmd.Name = command.PolicyCheck
```

Comment:
cmd is a pointer I think this is not what we want. This is what i mentioned to you before.